src/Native/Runtime/PalRedhawkCommon.h

```diff
@@ -112,6 +112,15 @@ struct PAL_LIMITED_CONTEXT
     void SetIp(UIntNative ip) { IP = ip; }
     void SetSp(UIntNative sp) { Rsp = sp; }
     UIntNative GetFp() const { return Rbp; }
+#elif defined _TARGET_WASM_
```

Comment:
Change this to `#elif defined(_TARGET_X86_) || defined(_TARGET_AMD64_)` and move the dummy implementation with `PORTABILITY_ASSERTs` into the #else block.

Reply:
Done